test/inductor/test_torchinductor.py

```diff
@@ -72,6 +72,7 @@
     parametrize,
     skipIfRocm,
     skipIfRocmArch,
+    subtest,
```

Comment:
Why this is required?

Reply:
This import was already there, it was accidentally removed by this commit:

https://github.com/ROCm/pytorch/commit/0b08ab6eeffd5d1ad34bf74affc810ae23542992#diff-809c39aeafb3acc92289f42a63e670a8719d4ce5627d5f88820142d80edf8d2aL74